source/Ship.cpp

```diff
@@ -2009,7 +2011,12 @@ void Ship::DoGeneration()
 	
 	heat -= heat * HeatDissipation();
 	if(heat > MaximumHeat())
-		isOverheated = true;
+		if(cookable && heat > MaximumHeat()*1.2) {
+			hull = MinimumHull() - 1;
```

Comment:
Cooking is a process, not instantaneous.
Instead of a step change in the hull value at some magic heat level, the hull should decrease proportionally to the excess heat.

(Also, unconditionally setting the hull to some value just below the minimum could _repair_ the hull when taking heat damage, if the hull is more damaged than that value.)

Reply:
The point is the organism inside the shell dies, not the hull taking damage but there is no way to permanently disable a ship without lowering its hull below the minimum as far as I know

Reply:
Hull is used more as a "representation of the health of the total ship/creature" than as a reference to just the outer shell.
We don't differentiate in the game between ships that have all internal systems blown out (what you would expect with piercing ammunition) and ships that have the outer hull blown to pieces. (All ships that get destroyed act like the latter and explode as if they were fully blown to pieces.)

Reply:
Done